frontend/src/metabase/modes/components/drill/SummarizeColumnByTimeDrill.js

```diff
@@ -50,6 +51,8 @@ export default ({
               ? [aggregator.short, fieldRefForColumn(column)]
               : [aggregator.short],
           )
-          .pivot([dateDimension.defaultDimension().mbql()]),
+          .pivot([
+            DatetimeFieldDimension.defaultDimension(dateDimension).mbql(),
```

Comment:
Actually, I think you should have left this like you had it using the instance method, which should call the static method.